docs/user/03-11-customize_secret.md

```diff
@@ -0,0 +1,62 @@
+# Customize the Default Credentials and Access
+
+> [!NOTE]
+> This feature is still being developed and will be available soon.
+
+You can customize the `sap-btp-manager` Secret and manage your own default configuration of the SAP BTP Operator module.
+
+## Context
+
+When you create SAP BTP, Kyma runtime, the `sap-btp-manager` Secret is automatically created as the default Secret managing the SAP BTP Operator's resources. 
+Because of Kyma's automatic reconciliation, any changes to the Secret are reverted, and the previous settings are restored within up to 24 hours.
+See [Preconfigured Credentials and Access](03-10-preconfigured-secret.md#credentials).
+
+To customize the `sap-btp-manager` Secret and prevent your changes from being reverted, you must stop the Secret's reconciliation.
+With the customized Secret, you can perform the following actions:
+
+* Manage your own default configuration of SAP BTP Operator
+* Migrate the service instances you created outside of the Kyma environment to your Kyma cluster
+
+## Procedure
+
+To customize the `sap-btp-manager` Secret, perform the following steps:
```

Comment:
The world "steps" made me think that I need to conduct steps in specific order but it is more about remembering to do the following modification to the secret. Could you rephrase "... perform the following steps" to something like "... perform the following secret modifications"?

Reply:
> To customize the `sap-btp-manager` Secret, perform the following secret modifications.
Sounds a bit awkward almost like 
> To modify the `sap-btp-manager` Secret, perform the following secret modifications.

On the other hand the order is not relevant provided we do all modifications during single edit operation (kubectl edit). 
But we are not sure how user will proceed so it is safer to impose order beginning with setting the label to prevent reconciliation.

So I would vote to leave it as it is.

Reply:
I'll rephrase the sentence if it causes confusion, but mind the fact that we can have ordered steps for procedures when the order of performing instructions is essential and unordered steps for cases like this one when you must perform them all in any order.

Reply:
I understood that the document follows a [Task Topic](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/00-content-strategy.md#task-topics) structure. As I understand there would be only one step in this document: "create a secret" and this is just a description of that step and the order does not matter in that paragraph.